.azure-pipelines/bazel.yml

```diff
@@ -67,9 +67,7 @@ steps:
       echo "disk space at end of build:"
       df -h
       # Cleanup offending files with unicode names
-      rm -rf ${ENVOY_DOCKER_BUILD_DIR}/*/*/external/go_sdk/test/fixedbugs
-    env:
-      ENVOY_DOCKER_BUILD_DIR: $(Build.StagingDirectory)
+      rm -rf $(Build.StagingDirectory)/tmp/*/*/external/go_sdk/test/fixedbugs
```

Comment:
Whoa, how did you manage to push to my fork branch?

Reply:
If you checked "allow edits from maintainers": https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork